ddl/util/util.go

```diff
@@ -204,6 +212,78 @@ func LoadGlobalVars(ctx context.Context, sctx sessionctx.Context, varNames []str
 	return nil
 }
 
+const (
+	newSessionRetryInterval = 200 * time.Millisecond
+	logIntervalCnt          = int(3 * time.Second / newSessionRetryInterval)
+
+	// NewSessionDefaultRetryCnt is the default retry times when create new session.
+	NewSessionDefaultRetryCnt = 3
+	// NewSessionRetryUnlimited is the unlimited retry times when create new session.
+	NewSessionRetryUnlimited = math.MaxInt64
+)
+
+// NewSession creates a new etcd session.
+func NewSession(ctx context.Context, logPrefix string, etcdCli *clientv3.Client, retryCnt, ttl int) (*concurrency.Session, error) {
```

Comment:
Why did we put this function here? This function is not necessarily related to DDL, as it is also used in "info.go"?

Reply:
well, `tidb/util` is a better place I think?

Reply:
If you want to put it on `util`, I think so. I'm not sure why we need to remove it from `owner`. You can also describe this in PR

Reply:
it will be cycle import after implementing `requireOwner`.

Reply:
Got it. It can be explained in PR, otherwise, others may not be clear about it.